resources/kcp/charts/provisioner/templates/gardener-creds-secret.yaml

```
@@ -1,9 +1,11 @@
-{{- if eq .Values.gardener.manageSecrets true }}
+{{- if eq .Values.gardener.manageSecrets false }}
```

This must not be changed, we need to provide an override for this value in config :)
```suggestion
{{- if eq .Values.gardener.manageSecrets false }}
```